test/nix/modules/default.nix

```diff
@@ -154,6 +157,7 @@ forEachUbuntuImage
               node1.succeed("${newConfig}/bin/deactivate")
               node1.fail("systemctl status new-service.service")
               node1.fail("cat /etc/foo_new")
+              node1.fail("test -f /var/tmp/system-manager/foo1")
```

Comment:
We should also test here that tmpfiles of the host distro still exist at this point.